src/Magento/FunctionalTestingFramework/Util/Script/ScriptUtil.php

```diff
@@ -98,15 +102,34 @@ public function getModuleXmlFilesByScope($modulePaths, $scope)
         return $found ? $finder->files() : [];
     }
 
+    /**
+     * Return suite XML files in TESTS_BP/ROOT_SUITE_DIR directory
+     *
+     * @return Finder|array
+     * @throws TestFrameworkException
+     */
+    public function getRootSuiteXmlFiles()
```

Comment:
couldn't find any usages of this function. Also, are we calling out suites which have references to deprecated tests as a part of this static-checks?